tock/tock/templates/base.html

```diff
@@ -121,5 +121,9 @@
       {% block js %}
       {% endblock js %}
 <div>
+  <footer>
+    <p><i>The current release is <code>{{ release_name }}</code>, <a href="{{ release_url }}" title="{{ release_url }}">view this release on GitHub</a></i>.</p>
```

Comment:
Reads like,

_The current release is `v20180202.1`, [view this release on GitHub](https://github.com/18F/tock/releases/tag/v20180202.1)_.

Reply:
That `div` wrapping footer is probably redundant, since `footer` can fulfill the same role in a more semantic way. If you do need it for some reason, I don't think it's currently closed.

Also, I'd double-check on this, but I think that `title` attribute may be hurting your accessibility.

Reply:
The `title=""` attribute seems to neither be hurting nor helping my accessibility here. Though my age may be showing here, I just like having the hover tooltip when mouseover the link 😄.

Good catch on the `<div>`. I think it's supposed to be a closing tag. Looks like it's been that way for a while 😆.

Reply:
Here's some info on title attributes and a11y: https://developer.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/
https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/

I'm not sure if yours is a problem or not. It's probably not a big deal, but I'd suggest a look on those links, and maybe double check in #g-accessibility